src/utilities/MessageFormatter.ts

```diff
@@ -24,41 +24,16 @@ const formatMessageData = (message: MessageProps, userInfo: IUserInfo) => {
 };
 
 const formatEncryptedMessageData = async (
-  message: MessageProps,
-  userName: string
+  text: string,
+  conversationId: string
 ) => {
-  return generateKey('Arnold', 'salt', 5000, 256).then((key) => {
-    return decryptData(message.text, key)
+  return generateKey(conversationId, 'salt', 5000, 256).then((key) => {
```

Comment:
you need to add the 'salt', 5000, 256 to default value

Reply:
ok I updated pass optionAEScrypto prop from outside package